locales/bn/bn.json

```diff
@@ -689,14 +689,14 @@
     "Welcome Back!": "স্বাগতম!",
     "Western Sahara": "পশ্চিমা সাহারা",
     "When two factor authentication is enabled, you will be prompted for a secure, random token during authentication. You may retrieve this token from your phone's Google Authenticator application.": "প্রমাণীকরণ সক্রিয় করা হয় দুই ফ্যাক্টর, আপনি অনুমোদনের সময় একটি নিরাপদ, র্যান্ডম টোকেন জন্য অনুরোধ করা হবে. আপনি আপনার ফোন এর গুগল প্রমাণকারী অ্যাপ্লিকেশন থেকে এই টোকেন উদ্ধার হতে পারে.",
```

Comment:
Is the correct character used to end a sentence?

In other places, I noticed the use of the vertical slash character (`|`), but here - a period (`.`).

This translation was by machine.